int64 argumento1
int64 argumento2
---
int64 resultado1